364.json

```diff
@@ -0,0 +1,21 @@
+{
+  "id": 364,
+  "created_at": "2018-01-24",
+  "updated_at": "2018-01-24",
+  "title": "Fastify denial-of-service vulnerability with large JSON payloads",
+  "author": "nwoltman (https://github.com/nwoltman)",
+  "module_name": "fastify",
+  "publish_date": "2018-01-24",
+  "cves": [
+    "CVE-2018-3711"
+  ],
+  "vulnerable_versions": "<=0.37.0",
+  "patched_versions": ">0.38.0",
+  "slug": "fastify_remote-dos",
+  "overview": "Fastify prior to 0.37.0 is vulnerable to a denial-of-service attack by sending a request with Content-Type set to application/json and a very large payload.",
+  "recommendation": "Update to version 0.38.0 or later.",
+  "references": "- MISC:https://github.com/mqttjs/MQTT.js/commit/403ba53b838f2d319a0c0505a045fe00239e9923\n- MISC:https://github.com/mqttjs/MQTT.js/releases/tag/v2.15.0\n- JVN:JVN#45494523\n- URL:https://jvn.jp/en/jp/JVN45494523/index.html",
```

Comment:
@lirantal can you add a link to the public report on H1 ?

Reply:
this link is wrong.

Reply:
yep, thanks.